src/sip.c

```diff
@@ -615,6 +634,8 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
         if (!strncmp(line, "c=", 2)) {
             if (sscanf(line, "c=IN IP4 %s", dst.ip) && media) {
                 media_set_address(media, dst);
+                strcpy(rtp_stream->dst.ip, dst.ip);
+                strcpy(rtcp_stream->dst.ip, dst.ip);
```

Comment:
The lack of this lines generating incomplete streams on the call, since the stream info was not being updated or set.